bot/exts/backend/error_handler.py

```diff
@@ -245,7 +245,6 @@ async def handle_user_input_error(self, ctx: Context, e: errors.UserInputError)
         elif isinstance(e, errors.ArgumentParsingError):
             embed = self._get_error_embed("Argument parsing error", str(e))
             await ctx.send(embed=embed)
-            self.get_help_command(ctx).close()
```

Comment:
Dont't we still need to remove the trashcan? Though that may be different..

Reply:
This is never called, it's closed immediately after creation. This is an artifact left behind from when the function had a slightly different structure.